src/lib/getIgnoredUpgradesDueToEnginesNode.ts

```diff
@@ -0,0 +1,49 @@
+import { minVersion, satisfies } from 'semver'
+import { IgnoredUpgradeDueToEnginesNode } from '../types/IgnoredUpgradeDueToEnginesNode'
+import { Index } from '../types/IndexType'
+import { Maybe } from '../types/Maybe'
+import { Options } from '../types/Options'
+import { Version } from '../types/Version'
+import { VersionSpec } from '../types/VersionSpec'
+import getEnginesNodeFromRegistry from './getEnginesNodeFromRegistry'
+import upgradePackageDefinitions from './upgradePackageDefinitions'
+
+/** Checks if package.json min node version satisfies given package engine.node spec */
+const satisfiesNodeEngine = (enginesNode: Maybe<Version>, optionsEnginesNodeMinVersion: string) =>
```

Comment:
`enginesNode` should be `Maybe<VersionSpec>`, not `Maybe<Version>`, since it can be represented as a version range.

Reply:
done